### ISSUE TITLE
Fingerprint print scanner not working 

### ISSUE THREAD

**Issue by Athulnalin** [kdrag0n/safetynet-fix#177]
**Describe the bug**

**To reproduce**

**Expected behavior**

**Screenshots**

**Device info**
Device model:
Android version:
ROM name/version:

**Logs**
Connect your phone to a computer and run `adb logcat > issue.log`. Attach the log file to this issue.

**Additional context**

**Checklist**
- [ ] All information is present
- [ ] Logs are attached
- [ ] I have tried installing and configuring [MagiskHide Props Config]https://github.com/Magisk-Modules-Repo/MagiskHidePropsConf)
After installing the saftynet fix module the fingerprint scanner is not working.the saftynet test is passed now

**Comment by Jon8RFC** [comment]
You'll need to come back here and look at your report you made, and then fill in all of the blanks.

**Comment by JakobLichterfeld** [comment]
Maybe duplicate of #173

**Comment by frnode** [comment]
Hello everyone,

If you wish, I worked on the issue and now it works (fingerprint and SafetyNet verification), you can find the change below:
https://github.com/frnode/safetynet-fix/commit/dc3ec64eff578bc88aa6bb2970c026d71f9799e0?diff=split#diff-e39ee2cf32c559774a334de83bb183fcd361c35bdd40c208cd809e1915df3402dL111

I also compiled it, here: https://github.com/frnode/safetynet-fix/releases/tag/v2.3.0 
If you prefer, you can also compile it yourself, with the docker I have also added.

With pleasure ! 🚀

**Comment by frnode** [comment]
Hello everyone,

If you wish, I worked on the issue and now it works (fingerprint and SafetyNet verification), you can find the change below:
https://github.com/frnode/safetynet-fix/blob/099e939452cd024340f1091dc4f65c297bde7d89/zygisk/module/jni/module.cpp?plain=1#L134

I also compiled it, here: https://github.com/frnode/safetynet-fix/releases/tag/v2.3.0 
If you prefer, you can also compile it yourself, with the docker I have also added.

With pleasure ! 🚀

**Comment by ruszok** [comment]
Works fine! Thanks!

**Comment by frnode** [comment]
@adwil13 Yes, device with the latest C20 update, there is a problem with CTS verification. 
I will look soon and I will update the fork if I find the solution.

**Comment by ruszok** [comment]
+1 props works but then fingerprint doesn't.

**Comment by kdrag0n** [comment]
Fixed in [v2.3.1](https://github.com/kdrag0n/safetynet-fix/releases/tag/v2.3.1).

**Comment by kdrag0n** [comment]
> > Fixed in [v2.3.1](https://github.com/kdrag0n/safetynet-fix/releases/tag/v2.3.1).
> 
> Should both works - fingerprint and Safetynet? In my case FP works fine Safetynet not pass.

That's a different issue. #188
